geoprocessing-tools.qmd

```diff
@@ -151,20 +152,20 @@ tool_exec <- function(in_params, out_params) {
 ```
 
 :::{.callout-note}
-If you are going to filter or select columns from your dataset consider doing that with the `fields` and `where_clause` arguments of `arc.select()` to reduce the amount of data read into memory. 
+To filter or select columns from a dataset, consider using the `fields` and `where_clause` arguments of `arc.select()` to reduce the amount of data read into memory. 
 
-`arc.select()` returns a `data.frame` with fields and a special geometry column which is incompatible with sf. We use `arc.data2sf()` to convert it to an sf object.
+`arc.select()` returns a `data.frame` with fields and a special geometry column which is incompatible with sf. Use `arc.data2sf()` to convert it to an sf object.
 :::
 
 
 ### Writing a Feature Class
 
-It is quite common to write the results of an analysis to a file geodatabase. This can be done with `arc.write()`. Typically, the we will capture the output path to write to in the `out_params` list object as a Feature Class type parameter. 
+It is quite common to write the results of an analysis to a file geodatabase. This can be done with `arc.write()`. A Feature Class type parameter can be used in the `out_params` list object to capture a user-defined output path.
 
-`arc.write()` requires two arguments. The first is the output path to write to and the second is the object to write. The accepted type of objects are `data.frame`, `sf`, `RasterLayer` or `RasterBrick`. 
+`arc.write()` requires two arguments. The first is the output path to write to and the second is the object to write. The accepted types of objects are `data.frame`, `sf`, `RasterLayer` or `RasterBrick`. 
 
 :::{.callout-note}
-As of Nov 2023 `{terra}` is not supported but we are actively working on this.
+As of Nov 2023, support for `{terra}` is in development.
```

Comment:
Support for `{terra}` is planned to accompany the ArcGIS Pro version 3.3 release.